docs/flow/config-language/expressions/function_calls.md

```diff
@@ -6,3 +6,25 @@ weight: 400
 ---
 
 # Function Calls
+Function calls is one more River feature that lets users build richer
+expresisons.
+
+Functions take zero or more arguments as their input and always return a single
+value as their output. Functions cannot be constructed by users, but can be
+either called from River's standard library, or exported by a compoonent.
+
+## Standard library functions
+River contains a [standard library][] of useful functions. Some enable
+interaction with the host system (eg. reading from an environment variable), or
+allow for more complex expressions (eg. concatenating arrays or decoding JSON
+strings into objects).
+```river
+env("HOME")
+json_decode(local.file.cfg.contents)["namespace"]
+```
+
+## Exported by components
+Components can also expose functions in their named Exports, as documented on
+their reference page. 
```

Comment:
Since we don't have any components that do this yet, should we remove this section?

Reply:
Yeah, I removed the dedicated section but left the mention of the feature in the overview paragraph above.